package exceptions;

public class Exception1 {

    public static void main(String[] args){


    }

    public static void demoArrayException(){

        String[] data = {"s1","s2","s3","s4"};
        System.out.println("The result is:" +data);
    }
}
